modular_skyrat/modules/cryosleep/code/cryopod.dm

```diff
@@ -134,7 +134,11 @@ GLOBAL_LIST_EMPTY(cryopod_computers)
 	///Weakref to our controller
 	var/datum/weakref/control_computer_weakref
 	COOLDOWN_DECLARE(last_no_computer_message)
+	var/quiet = FALSE
```

Comment:
code doc comment this (`/// what you want the comment to be` above the `var/quiet` code)

Reply:
Done! Code modified as requested!